### DISCUSSION TITLE
Open sidebar tabs as a full new tab

### DISCUSSION THREAD

**Discussion by vickygamer997** [Floorp-Projects/Floorp#862] [Feature Request]
Just getting the URL from the pages is a hassle. I'll also note that Edge has a similar feature with its side bar.

**Comment by uugaabuugaa** [comment]
Came to ask for THIS. Becomes a chore when I have to type the url or copy paste a url after opening a web panel and putting it to new tab.

**Comment by SystemError12** [comment]
You mean with its own adress bar? Not quite getting what for. Reasonable thing maybe would be to add a Right-Click option to show or copy the URL.. . 

Personally I prefer sidebar and tabs being different. I can put whichever web-site I like to what suits it best and have options. If sibebar is just another tab, that would just make it as another bookmark-toolbar, which we already can use if we need it below the adress bar.

**Reply by vickygamer997** [reply]
No, I mean with a button that will create a new tab that opens to the page you're currently on in the sidebar.

**Comment by uugaabuugaa** [comment]
I think you're right about it becoming a bookmark toolbar. I don't use bookmarks as a frequently visiting site; just to keep tabs of sites I don't wanna lose. But it's not like I'm always going from sidebar to new tab. Lets say I use gmail on the sidebar to check if anything important came up and nothing does most of the time. That one out of ten times when it does I'd open it on a new tab to zone in and reply. So just an option to open it up in another tab is really efficient in my opinion. And I don't think a right click option to just see or copy the url is any better than just bookmarking it.

**Comment by SystemError12** [comment]
That might be a good thing for some, for those who dont use it adding too many options will only clutter the browser with unnecessary functions. I guess a [customizable Right-Click Menu](https://github.com/Floorp-Projects/Floorp/discussions/1013), that i suggested in the other Post, could possibly be the solution for this as well, so whoever can make use of that function can just customize it.

**Reply by vickygamer997** [reply]
I don't think one more button is a lot. It's just a button.

**Reply by SystemError12** [reply]
Ok I understand now what you mean. I can understand that this option can be useful, so generally I am in favor of it. But I would prefer if they keep the actual tab clean and don't add another button within the tab. I would much rather just add it as a right-click option (right click on the favicon or somewhere on the upper edge). Something like "Open in new tab" would defenitely be a good addition to the sidebar and "nice to have".  

Even better if they can manage to make the right-click menu customizable as I wrote before. So people can chose what function they want there and what not. But I can understand that this is much harder (even if worthwhile in the end) thing to realize with all the Settings-Options you would need to add, than just 1 more single option.

**Comment by uugaabuugaa** [comment]
Yes of course. The more customization available the better!
